tests/tensor_base/test_tensor.py

```diff
@@ -243,7 +256,7 @@ def test_neg(x):
     x=hnp.arrays(
         shape=hnp.array_shapes(),
         dtype=hnp.floating_dtypes(),
-        elements=st.floats(-10, 10),
+        elements=st.floats(-10, 10, width=16),
```

Comment:
why?

Reply:
Because I am testing for varying float-widths, the smallest being 16-bit. Hypothesis will correctly complain if I generate wider floats but then cast them down to a smaller width - it can violate some assumptions made in the strategy.

Reply:
I'm only seeing float16 here? The only other place I see width being used is with

``` python
width=8 * np.dtype(numpy_dtype).itemsize
```

which makes sense

Reply:
```python
        dtype=hnp.floating_dtypes(),  # <--- here
        elements=st.floats(-10, 10, width=16),
```

Reply:
ah! Yes I should use my eyes when I read